pillarbox-demo/src/main/java/ch/srgssr/pillarbox/demo/ui/player/SimplePlayerViewModel.kt

```diff
@@ -45,6 +63,13 @@ class SimplePlayerViewModel(application: Application) : AndroidViewModel(applica
     var pictureInPictureRatio = MutableStateFlow(Rational(1, 1))
 
     init {
+        notificationManager = PlayerNotificationManager.Builder(application, NotificationId, "Hello")
```

Comment:
Shouldn't we use something else for the channel id, rather than `Hello`?

Reply:
Yes right, I will set the one we had before.